website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -61,7 +59,7 @@ You can use the [deployed contracts for a released version][doc-released-contrac
 [IRiscZeroVerifier]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/src/IRiscZeroVerifier.sol
 [RiscZeroVerifierRouter.sol]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/src/RiscZeroVerifierRouter.sol
 [VersionManagement]: https://github.com/risc0/risc0-ethereum/blob/main/contracts/version-management-design.md
-[doc-released-contracts]: /api/blockchain-integration/contracts/verifier#contract-addresses
+[doc-released-contracts]: /api/blockchain-integration/contracts/verifier.md#contract-addresses
```

Comment:
I was under the impression that absolute links can't be filenames. This might break the link (unsure).

Reply:
mhmm I think you are correct -- will remove this logic for absolute links 👍